src/ceph-volume/ceph_volume/devices/lvm/batch.py

```diff
@@ -18,87 +20,82 @@
-                                     osd_id,
-                                     'dmcrypt' if args.dmcrypt else None,
-                                     dev.symlink))
-    return ret
-
-
-def get_lvm_osds(lvs, args):
-    '''
-    Goes through passed LVs and assigns planned osds
-    '''
-    ret = []
-    for lv in lvs:
-        if lv.used_by_ceph:
-            continue
-        osd_id = None
-        if args.osd_ids:
-            osd_id = args.osd_ids.pop()
-        osd = Batch.OSD("{}/{}".format(lv.vg_name, lv.lv_name),
-                        100.0,
-                        disk.Size(b=int(lv.lvs[0].lv_size)),
-                        1,
-                        osd_id,
-                        'dmcrypt' if args.dmcrypt else None)
-        ret.append(osd)
-    return ret
-
-
-def get_physical_fast_allocs(devices, type_, fast_slots_per_device, new_osds, args):
+# def get_physical_osds(devices: List[device.Device], args: argparse.Namespace) -> List[Batch.OSD]:
+#     '''
+#     Goes through passed physical devices and assigns OSDs
+#     '''
+#     data_slots = args.osds_per_device
+#     if args.data_slots:
+#         data_slots = max(args.data_slots, args.osds_per_device)
+#     rel_data_size = args.data_allocate_fraction / data_slots
+#     mlogger.debug('relative data size: {}'.format(rel_data_size))
+#     ret = []
+#     for dev in devices:
+#         if dev.available_lvm:
+#             dev_size = dev.vg_size[0]
+#             abs_size = disk.Size(b=int(dev_size * rel_data_size))
+#             free_size = dev.vg_free[0]
+#             for _ in range(args.osds_per_device):
+#                 if abs_size > free_size:
+#                     break
+#                 free_size -= abs_size.b
+#                 osd_id = None
+#                 if args.osd_ids:
+#                     osd_id = args.osd_ids.pop()
+#                 ret.append(Batch.OSD(dev.path,
+#                                      rel_data_size,
+#                                      abs_size,
+#                                      args.osds_per_device,
+#                                      osd_id,
+#                                      'dmcrypt' if args.dmcrypt else None,
+#                                      dev.symlink))
+#     return ret
+
+
+# def get_lvm_osds(lvs: List[Volume], args: argparse.Namespace) -> List[Batch.OSD]:
+#     '''
+#     Goes through passed LVs and assigns planned osds
+#     '''
+#     ret = []
+#     for lv in lvs:
+#         if lv.used_by_ceph:
+#             continue
+#         osd_id = None
+#         if args.osd_ids:
+#             osd_id = args.osd_ids.pop()
+#         osd = Batch.OSD("{}/{}".format(lv.vg_name, lv.lv_name),
+#                         100.0,
+#                         disk.Size(b=int(lv.lvs[0].lv_size)),
+#                         1,
+#                         osd_id,
+#                         'dmcrypt' if args.dmcrypt else None)
+#         ret.append(osd)
+#     return ret
```

Comment:
We should avoid leaving commented out code in the final patches, JFYI.

Reply:
@phlogistonjohn , for what it's worth, this PR isn't ready to be merged.
This code is not supposed to be left commented out.

Reply:
OK, no problem. The PR wasn't draft until recently so I was treating it like it was ready for full review.

Reply:
reviews are still more than welcome in any case 😄